src/service/iplayerService.ts

```diff
@@ -428,7 +428,7 @@ async function createResult(term: string, details: IPlayerDetails, sizeFactor: n
         episode: details.episode,
         pubDate: details.firstBroadcast ? new Date(details.firstBroadcast) : undefined,
         series: details.series,
-        type: VideoType.TV,
+        type,
```

Comment:
Noticed this may cause issues as the code above does allow for movies to hit this, but then it always returned a TV show.